com.endlessm.apps.Sdk.json.in.in

```diff
@@ -208,7 +208,12 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "https://github.com/endlessm/xapian-core"
+                    "url": "https://github.com/endlessm/xapian-core",
+                    "branch": "xapian-next"
```

Comment:
This should be removed now that the other PRs are merged.

Reply:
If we do that, we're going to build xapian 1.3, as that's what the master branch points to.

We should have a xapian-endless-1.3 branch that we use for SDK < 4, and then track a specific commit on the upstream master branch.

Reply:
We discussed this during standup: ignore this comment, I was confused between `xapian-core` and `xapian-glib`. However, since we don't have any distro patches anymore, we can just use the upstream repo and pin it to the commit ID that Olly gives us.